requirements.txt

```diff
@@ -2,6 +2,6 @@ RadeonOpenCompute/rocm-cmake@04f694df2a8dc9d7e35fa4dee4ba5fa407ec04f8 --build
 sqlite3@3.17 -DCMAKE_POSITION_INDEPENDENT_CODE=On
 boost@1.79 -DCMAKE_POSITION_INDEPENDENT_CODE=On --build
 half,https://github.com/pfultz2/half/archive/1.12.0.tar.gz -X header -H sha256:0a08660b68abb176ebc2a0cdf8de46e3182a7f46c66443bb80dbfaaec98cf969 --build
-ROCmSoftwarePlatform/llvm-project-mlir@rocm-5.3.0 -H sha256:ee5093aad5459773c350205ef937a186a20994b42798106f8126b9914ad099a5 -DBUILD_FAT_LIBMLIRMIOPEN=1
+ROCmSoftwarePlatform/rocMLIR@release/rocm-5.4  -DBUILD_FAT_LIBROCKCOMPILER=1
 nlohmann/json@v3.9.1 -DJSON_MultipleHeaders=ON -DJSON_BuildTests=Off
-ROCmSoftwarePlatform/composable_kernel@7589116121f80189f47cfd8692f300ee8c6377ad -D CMAKE_CXX_FLAGS=" --offload-arch=gfx900 --offload-arch=gfx906 --offload-arch=gfx908 --offload-arch=gfx90a --offload-arch=gfx1030 -O3 " 
+ROCmSoftwarePlatform/composable_kernel@7589116121f80189f47cfd8692f300ee8c6377ad -D CMAKE_CXX_FLAGS=" --offload-arch=gfx900 --offload-arch=gfx906 --offload-arch=gfx908 --offload-arch=gfx90a:xnack- --offload-arch=gfx1030 -O3 " 
```

Comment:
@JehandadKhan Looks like this `--offload-arch` has been blocking @zhanglx13 for weeks. It looks like a composable kernel repo's particular concept. Could you help explain what this is? (and why this is not caught by previous docker image builds?)